include/dlaf/blas/tile.h

```diff
@@ -191,6 +237,16 @@ void herk(cublasHandle_t handle, const blas::Uplo uplo, const blas::Op op, const
                                 util::blasToCublasCast(&beta), util::blasToCublasCast(c.ptr()), c.ld());
 }
 
+/// Performs a matrix-matrix multiplication, involving a triangular matrix.
+template <class T>
+void trmm(const blas::Side side, const blas::Uplo uplo, const blas::Op op, const blas::Diag diag,
+          const T alpha, const Tile<const T, device>& a, const Tile<T, device>& b) noexcept {
+  auto s = tile::internal::getTrmmSizes(side, op, a, b);
+  blas::CublasTrmm(util::blasToCublas(side), util::blasToCublas(uplo), util::blasToCublas(op),
```

Comment:
Are you testing these new wrappers?

I'm asking because I just gave a quick look, and I think this is not going to compile.
Shouldn't this look more something like `internal::CublasTrmm<T>::call`?

Reply:
I think you're right. I've fixed it. 
I don't have experience with CUDA actually: I've just tried to copy the implementation of the other wrappers. How can I test them? Thanks.

Reply:
They'd go in `test_cublas_tile.cpp` and `test_cublas_tile/test_trmm.h`. The existing tests *should* be pretty straightforward to adapt to `trmm`, but if you have any questions, please don't hesitate to ping me. I can help you with those as well.

Reply:
Thank you @msimberg! I'll try to adapt it: I'll reach out in case I have questions.

Reply:
@msimberg, if you have time, can you please check if it is ok? Thanks!